tests/interaction.test.js

```
@@ -0,0 +1,180 @@
+import {
+  difference,
+  fusion,
+  intersect,
+  assembly,
+  loftShapes,
+  shrinkWrapSketches,
+  is3D,
+} from "../src/worker/interaction.js";
+import { circle, rectangle } from "../src/worker/shapes.js";
+import { extrude } from "../src/worker/actions.js";
+import { init, is3D } from "../src/worker/util.js";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'is3D'. Consider to delete it or rename it.

'is3D' is defined here:


(lint/suspicious/noRedeclare)